airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -66,36 +76,77 @@ def request_params(self, **kwargs) -> MutableMapping[str, Any]:
 
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
         response_json = response.json()
-        yield from response_json.get(self.data_field, [])
+        records = response_json.get(self.data_field, [])
 
+        for record in records:
+            yield self._convert_timestamp_fields_to_datetime(record)
 
-class IterableExportStream(IterableStream, ABC):
-    def __init__(self, start_date, **kwargs):
-        super().__init__(**kwargs)
-        self._start_date = pendulum.parse(start_date)
-        self.stream_params = {"dataTypeName": self.data_field}
+    @staticmethod
+    def _parse_plain_text_to_dict(text: str) -> Dict[str, Any]:
```

Comment:
not sure I understand what is going on here, could you add a bit more context? maybe an example of input and output?

Reply:
We get a csv text as a payload here. Updated